### PR TITLE
support for fixity checking

### PR DESCRIPTION
**Pull request by acoburn**
See: https://jira.duraspace.org/browse/FCREPO-1587

### CODE REVIEW

File: src/main/java/org/fcrepo/camel/RdfNamespaces.java [hunk 1]
```diff
@@ -32,6 +32,8 @@
 
     public static final String LDP = "http://www.w3.org/ns/ldp#";
 
+    public static final String PREMIS = "http://www.loc.gov/premis/rdf/v1#";
```
Comment (awoods) [inline src/main/java/org/fcrepo/camel/RdfNamespaces.java]: The PREMIS namespace appears to only be used in tests. Is this addition needed or useful in `RdfNamespaces`?

**Comment by acoburn** [comment]
@awoods it's the same as the `rdf` and `indexing` namespaces. They're just there as convenience values. I'm not particularly attached to them, but they are certainly convenient.

**Comment by awoods** [comment]
Let's leave it be.
